tutorials/levels.md

```diff
@@ -78,7 +78,7 @@ being added and removed.
 Take a look at the 2D example below. This example focuses on a single performer,
 but the same logic can be extended to multiple performers.
 
-<img src="https://bytebucket.org/ignitionrobotics/ign-gazebo/raw/9e88ca13a2b3b51a1b39abeb2b5b8db23f2e74e3/doc/architecture_design/01.png"/>
+<img src="https://raw.githubusercontent.com/ignitionrobotics/ign-gazebo/d62080da95edbb172c47eac883ec4b707b59bb38/doc/architecture_design/01.png"/>
```

Comment:
Should these image url's link to the master branch?

Reply:
Yeah I think that would be ideal, but these images have been removed from `default` a long time ago, maybe by mistake. So for now I just found a valid commit and used that instead of committing the image too, not to make this PR longer. But I can add it here...

Reply:
Gotcha. Also, didn't know bytebucket == bitbucket :smile: